src/ucp/core/ucp_ep.h

```diff
@@ -66,6 +66,7 @@ enum {
                                                         can be replaced with looking at local ID */
 
     /* DEBUG bits */
+#if ENABLE_DEBUG_DATA || UCS_ENABLE_ASSERT
```

Comment:
why needed?
can we change ucp_ep_update_flags() func to handle this, instead of defining all debug flags twice?

Reply:
removed